src/hotspot/share/adlc/output_c.cpp

```diff
@@ -527,9 +543,8 @@ static int pipeline_res_mask_initializer(
     // "0x012345678, 0x012345678, 4294967295"
     char* args = new char [36 + 1];
 
-    int printed = sprintf(args, "0x%x, 0x%x, %u",
-      resources_used, resources_used_exclusively, element_count);
-    assert(printed <= 36, "overflow");
+    snprintf_checked(args, 37, "0x%x, 0x%x, %u",
```

Comment:
Instead of 37, consider `36 + 1` to be clearly from the line above.  Or give that value a name.

Reply:
Updated to use 36 + 1.